### DISCUSSION TITLE
How to combine multiple drawers depending on a specific content but keeping them on the same side ?

### DISCUSSION THREAD

**Discussion by juslin03** [saadeghi/daisyui#709] [Q&A] [answered]
Hello everyone !
I'm trying to combine multiple drawers depending on specific content. How can it be possible like in the video below ?
Just an example with statics contents and I'll adapt to my content.

Need a help please 

https://user-images.githubusercontent.com/36005325/163163984-0cdef901-ded7-44cf-a22c-3d6e6b0ea15a.mp4

**Comment by saadeghi** [comment, marked as answer]
You can put 2 drawers inside each other like this: https://play.tailwindcss.com/MRvuTbNOB8  
But that would be ugly code and confusing.  

I would use 1 drawer and dynamically change the sidebar content using JS.

Another option is to use [`modal`](https://daisyui.com/components/modal/) : https://play.tailwindcss.com/JGnhJhMOrG

**Reply by juslin03** [reply]
I'm trying it sir and back shortly 😊😊

**Reply by juslin03** [reply]
I got it work now.
Thank you so much
